.github/workflows/build_and_deploy.yml

```
@@ -362,11 +363,17 @@ jobs:
       - name: Create XCFramework
         run: |
           mkdir -p "artifact/${{ env.ASSET_NAME }}"
+          mkdir -p "Headers-sim"
```

勘違いしてしまう人を防止するために、ちょっとした意図をコメントで書いておくのはどうでしょう 👀 
（まあ僕が鈍感すぎる気がしないでもないですが･･････。）

```suggestion
          # 必要なファイルだけコピー
          mkdir -p "Headers-sim"
```

---

なるほどです！
Suggestionありがとうございます🙏